### PR TITLE
Apf 1230 servicenow hub connector

### PR DESCRIPTION
**Pull request by SudharsanThumattiSathyamoorthy**

### CODE REVIEW

File: connectors/hub-servicenow/src/main/java/com/vmware/connectors/hub/servicenow/HubServiceNowController.java [hunk 1]
```diff
@@ -0,0 +1,534 @@
...
+            path = "/cards/requests",
+            produces = MediaType.APPLICATION_JSON_VALUE,
+            consumes = MediaType.APPLICATION_JSON_VALUE
+    )
+    public Mono<Cards> getCards(
+            @RequestHeader(AUTH_HEADER) final String auth,
+            @RequestHeader(BASE_URL_HEADER) final String baseUrl,
+            @RequestHeader(ROUTING_PREFIX) final String routingPrefix,
+            final Locale locale,
+            @AuthenticationPrincipal final String userEmail,
```
[the remark this answers is not in the document]
Comment (RyanBard) [reply connectors/hub-servicenow/src/main/java/com/vmware/connectors/hub/servicenow/HubServiceNowController.java]: How did you test any of this?

File: connectors/hub-servicenow/src/main/service/rhel/files/default-conf/application.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# vIDM public key url (you must override this in /etc)
```
Comment (RyanBard) [inline connectors/hub-servicenow/src/main/service/rhel/files/default-conf/application.properties]: woops :)

Comment (SudharsanThumattiSathyamoorthy) [reply connectors/hub-servicenow/src/main/service/rhel/files/default-conf/application.properties]: Somehow I have missed this. We can change as part of APF-1687

Comment (RyanBard) [reply connectors/hub-servicenow/src/main/service/rhel/files/default-conf/application.properties]: Yea I missed it too: https://github.com/vmware/connectors-workspace-one/blob/APF-1539_hub_concur_connector/connectors/hub-concur/src/main/service/rhel/files/default-conf/application.properties

**Pull request merged**